langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -99,6 +157,20 @@ public isolated function put(table<MapType> t, MapType val) = @java:Method {
 # It panics if parameter `val` has the same key as an existing member of parameter `t`,
 # or if parameter `val` is inconsistent with the inherent type of `t`.
 #
+# ```ballerina
+# table<Student> key(id) students = table [
```

Comment:
Doesn't have to be a table type descriptor with a key constraint, right? Wondering if it may make sense to have two examples.

Reply:
If we are going to use assert then it will make sense. In a case where we are going to add a new item to the table with an existing key.

Reply:
The plan is to include assertions, so we can assume that there'll be assertions.